collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java

```diff
@@ -465,9 +465,14 @@ public static void moveMediaFiles(String tempMediaPath, File formMediaPath) thro
         }
     }
 
-    public static void saveBitmapToFile(Bitmap bitmap, String path) {
+    public static void saveBitmapToFile(Bitmap bitmap, String path, Bitmap.CompressFormat... format) {
```

Comment:
Sorry I didn’t think of this before. Let’s derive the JPEG or PNG from the filename.

Reply:
done